addons/event/models/event_ticket.py

```diff
@@ -62,4 +62,76 @@ def _constrains_dates_coherency(self):
                 raise UserError(_('The stop date cannot be earlier than the start date.'))
 
     def _get_ticket_tz(self):
-        return self.event_type_id.use_timezone and self.event_type_id.default_timezone or self.env.user.tz
\ No newline at end of file
+        return self.event_type_id.use_timezone and self.event_type_id.default_timezone or self.env.user.tz
+
+    @api.model
+    def _get_event_ticket_fields_whitelist(self):
+        """ Whitelist of fields that are copied from event_type_mail_ids to event_mail_ids when
+        changing the event_type_id field of event.event """
+        return ['name']
+
+
+class EventTicket(models.Model):
+    _name = 'event.event.ticket'
+    _inherit = 'event.type.ticket'
+    _description = 'Event Ticket'
```

Comment:
Maybe add some docstring to explain that we inherit the template model ?
Or maybe not.
Or maybe yes?
Or maybe ignore me.
What is life?